app/build.gradle

```diff
@@ -45,5 +48,8 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
-
+    // Import the Firebase BoM, analytics, crashlytics
+    implementation platform('com.google.firebase:firebase-bom:26.3.0')
```

Comment:
1) create and import a keystore file in the project, which will be used currently for both debug and release builds, with the default credentials (ask me for details).
2) set the SHA certificate fingerprints in the Firebase for this app.